cutils.c

```diff
@@ -606,4 +608,57 @@ void rqsort(void *base, size_t nmemb, size_t size, cmp_f cmp, void *opaque)
     }
 }
 
+#if defined(_MSC_VER)
+ // From: https://stackoverflow.com/a/26085827
+int gettimeofday(struct timeval * tp, struct timezone * tzp)
```

Comment:
You should probably namespace this, otherwise it's a potential symbol clash for embedders. Also :-)
```suggestion
int gettimeofday(struct timeval *tp, struct timezone *tzp)
```